app/static/js/events/wizard/vue-app.js

```diff
@@ -350,10 +358,21 @@ function shouldDisableMove($this) {
         $this.event.start_time_time.trim() === '' ||
         $this.event.start_time_date.trim() === '' ||
         $this.event.end_time_time.trim() ==='' ||
-        $this.event.end_time_date.trim() === ''
+        $this.event.end_time_date.trim() === '' ||
+        checkTickets($this)
     );
 }
 
+function checkTickets($this) {
+    for( var ticket of $this.event.tickets){
+            if(ticket.type == 'paid' && (ticket.price === '' || ticket.price <=0))
+            {
```

Comment:
The logic is flawed. Even if one ticket is valid, the function exits and everything is considered valid.